gui/wxpython/gui_core/ghelp.py

```diff
@@ -819,6 +820,32 @@ def fillContentsFromFile(self, htmlFile, skipDescription=True):
             self.loaded = False
 
 
+class ManualPageHelpWindow(HelpWindow):
```

Comment:
What is the purpose of creating a derived class, why not to implement that directly in HelpWindow?

Reply:
I wrongly thought that the base classes are used by About GRASS GIS dialog. Fixed with https://github.com/OSGeo/grass/pull/3291/commits/8184c419d915eca0f4dff3b7d8e68cc8a16cfbd3.